### ISSUE TITLE
Inifinite loop when parsing invalid HTML

### ISSUE THREAD

**Issue by caugner** [paquettg/php-html-parser#179]
I found a website in the wild with the following invalid/incomplete HTML, and unfortunately it makes the parser run into an infinite loop:

```
<html>
<head>
    <meta http-equiv="refresh" content="5; URL=http://www.example.com">
    <meta http-equiv="cache-control" content="no-cache">
    <meta http-equiv="pragma" content="no-cache">
    <meta http-equiv="expires" content="0">
</head>
<
```

**Comment by Gicminos** [comment]
I think I am having the same problem trying to fetch this website: https://www.cdw.com/

**Comment by paquettg** [comment]
This has been fixed in 2.1.0.

Please let me know if you need anything further.

**Comment by PWalkow** [comment]
As 2.1.0 is still in dev mode is it possible to merge this bugfix to previous stable version? In that state unfortunately it's impossible to use it.
